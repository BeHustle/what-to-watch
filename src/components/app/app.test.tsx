import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

it(`Render empty offers screen`, () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});
