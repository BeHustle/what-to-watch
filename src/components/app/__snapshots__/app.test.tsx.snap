// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render empty offers screen 1`] = `
<h1>
  Testg
</h1>
`;
